eng/common-tests/ci.yml

```diff
@@ -0,0 +1,28 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
```

Comment:
It doesn't look like this pipeline triggered as part of your PR. Did you update the pipeline to point to the new file yet? We should probably also add eng/common-tests/ci.yml to the list of trigger paths.